client/packages/system/src/Patient/Encounter/DetailModal.tsx

```diff
@@ -35,7 +35,7 @@ type Encounter = Pick<
 export const EncounterDetailModal: FC = () => {
   const patientId = usePatient.utils.id();
   const t = useTranslation('patients');
-  const { current, setCurrent } = usePatientModalStore();
+  const { current, selectModal } = usePatientModalStore();
```

Comment:
perhaps should be `currentModal` if the action is now `selectModal`?
or just `modal`?
also - I would use `setModal` rather than `selectModal` because 
- it matches the usual naming of xxx and setXxx that we use
- to select something usually means picking from a list, where set usually means assigning
- in redux, the `select` are retrieval methods, using that for assigning is a little confusing

Reply:
thanks [139aad4](https://github.com/openmsupply/open-msupply/pull/528/commits/139aad4b272b85141e4c6c7907b1bf6f1d620043)